projects/1.21/assets/mffs/mffs/lang/zh_cn.json

```
@@ -0,0 +1,194 @@
+{ 
+  "itemGroup.mffs.main": "MFFS",
+   
+   "info.mffs.coercion_deriver.mode.integrate": "集成",
+   "info.mffs.coercion_deriver.mode.derive": "派生",
+   "info.mffs.link": "链接到 %s 的 %s",
+   "info.mffs.frequency": "频率：%s",
+   "info.mffs.frequency.generated": "生成的随机频率：%s",
+   "info.mffs.frequency.set": "频率设置为：%s",
+   "info.mffs.cannot_harness": "无法从 Fortron 领域获取 %s FE。",

```

```suggestion
   "info.mffs.cannot_harness": "无法从Fortron领域获取%s FE。",
```
一般情况下，英文字母、阿拉伯数字等非中文文本无需在前后添加空格。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **2. 格式规范 - 排版** 一节下。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。如有特殊需求请忽略。